internal/plan/plan.go

```diff
@@ -59,12 +59,13 @@ type Layer struct {
 
 type Service struct {
 	// Basic details
-	Name        string         `yaml:"-"`
-	Summary     string         `yaml:"summary,omitempty"`
-	Description string         `yaml:"description,omitempty"`
-	Startup     ServiceStartup `yaml:"startup,omitempty"`
-	Override    Override       `yaml:"override,omitempty"`
-	Command     string         `yaml:"command,omitempty"`
+	Name        string           `yaml:"-"`
+	Summary     string           `yaml:"summary,omitempty"`
+	Description string           `yaml:"description,omitempty"`
+	Startup     ServiceStartup   `yaml:"startup,omitempty"`
+	Override    Override         `yaml:"override,omitempty"`
+	Command     string           `yaml:"command,omitempty"`
+	GracePeriod OptionalDuration `yaml:"grace-period,omitempty"`
```

Comment:
`GracePeriod` should probably not be under the "Basic details" section. Move it to the "Auto-restart and backoff functionality" section?